README.md

```diff
@@ -172,8 +172,6 @@ Variant A would halt progress in the module graph until resolved.
 
 Variant B offers a unique approach to blocking, as it will not block siblings execution.
 
-Variant C would alleviate concerns of halting process
```

Comment:
nit: should this perhaps read `The Optional Constraint would alleviate concerns of halting process`?

Reply:
Updated ✨